lxd/networks.go

```diff
@@ -115,7 +115,8 @@ func networksPost(d *Daemon, r *http.Request) Response {
 		// value for the storage config is 'bridge.external_interfaces'.
 		for key := range req.Config {
 			if !shared.StringInSlice(key, db.NetworkNodeConfigKeys) {
-				return SmartError(fmt.Errorf("Invalid config key '%s'", key))
+				return SmartError(
+					fmt.Errorf("Config key '%s' may not be used as node-specific key", key))
```

Comment:
Can we avoid the wrapping here, makes grepping unnecessarily harder in this case (we don't have a max-width requirement in our codebase)?

Reply:
Done, hope it's what you meant.